hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/PrefixManagerImpl.java

```diff
@@ -145,14 +146,17 @@ public boolean checkAccess(OzoneObj ozObject, RequestContext context)
           return hasAccess;
         }
       }
+    } catch (IOException e) {
+      throw new OMException(e, INTERNAL_ERROR);
```

Comment:
Can we check here if the exception cause is InvalidPathException, 
and then turn it to an OMException with INVALID_PATH error code.

```
 new OMException(ex.getMessage(),
              OMException.ResultCodes.INVALID_PATH);
```

Reply:
Updated.